engine/app/views/citizens_advice_cookie_preferences/cookie_preferences/edit.html.erb

```diff
@@ -0,0 +1,79 @@
+<%= render CitizensAdviceComponents::PageContent.new do |c| %>
+  <% c.with_main do %>
+    <h1 class="cads-page-title">Your cookie settings</h1>
+
+    <div class="cads-prose">
+      <p> We’d like to add small text files called ‘cookies’ to your phone, tablet or computer when you use our website.</p>
+      <p> We use 3 types of cookies. They’re called:</p>
+      <ul>
+        <li>essential cookies</li>
+        <li>analytics cookies</li>
+        <li>video player cookies</li>
+      </ul>
+      <p> We’ll always add essential cookies to your device when you use our website. These cookies make sure our website works properly.</p>
+      <p> We’ll only add additional cookies if you give us permission. Additional cookies help us remember your preferences and show you things from other websites, like videos. They also help us find out how you use our website so we can keep improving it.</p>
+      <p>You can <a href="/about-us/information/how-we-use-cookies/">read detailed information about how we use cookies.</a></p>
+    </div>
+
+    <%= form_for @cookie_preferences, url: citizens_advice_cookie_preferences.cookie_preference_path, method: :patch do |form| %>
+      <div class="cads-form-field">
+        <fieldset class="cads-form-field__content cads-form-group cads-form-group--radio">
+          <legend>
+            <h2>Accept or reject analytics cookies</h2>
+          </legend>
+          <p>Analytics text here</p>
+          <p>You might not be able to:</p>
+          <ul>
+            <li>help us improve our website</li>
+          </ul>
+          <%= form.collection_radio_buttons :analytics, @cookie_preferences.analytics_cookies_options, :id, :name, required: true, label: "" do |b| %>
+            <div class="cads-form-group__item">
+              <%= b.radio_button(class: "cads-form-group__input") %>
+              <%= b.label(class: "cads-form-group__label") %>
+            </div>
+          <% end %>
+        </fieldset>
+      </div>
+
+      <div class="cads-form-field">
+        <fieldset class="cads-form-field__content cads-form-group cads-form-group--radio">
+          <legend>
+            <h2>Accept or reject video player cookies</h2>
+          </legend>
+          <p>Video player text here</p>
+          <p>You might not be able to:</p>
+          <ul>
+            <li>watch embedded videos</li>
+            <li>have your video player preferences remembered</li>
+          </ul>
+          <%= form.collection_radio_buttons :video_players, @cookie_preferences.video_players_cookies_options, :id, :name, required: true, label: "" do |b| %>
+            <div class="cads-form-group__item">
+              <%= b.radio_button(class: "cads-form-group__input") %>
+              <%= b.label(class: "cads-form-group__label") %>
+            </div>
+          <% end %>
+        </fieldset>
+      </div>
+
+      <%= form.cads_button "Save changes" %>
+
+      <div class="cads-prose">
```

Comment:
Question - could the `cads-prose` class wrap all the content apart from the `h1`? This should solve the issue of bulletpoint styling but I'm not sure it will add other spacing issues. There is also `.cads-prose-direct-descendants` which might be also worth trying in cases cads-prose needs to be applied to the fieldsets.